src/omero_zarr/cli.py

```diff
@@ -123,7 +126,9 @@ def _configure(self, parser):
             "--style",
             choices=("6d", "split", "labelled"),
             default="labelled",
-            help=("Choice of storage for ROIs"),
+            help=(
+                "Choice of storage for ROIs [breaks ome-zarr]"
```

Comment:
"Split" doesn't break ome-zarr (since it reads names of masks)? - I'll test to check...

Reply:
Yes - works for me (once I'd remembered to checkout the right branch!)

![Screenshot 2020-07-09 at 10 04 23](https://user-images.githubusercontent.com/900055/87020391-afca1600-c1cb-11ea-88ae-65609eeb9dbe.png)

Reply:
Were the splits registered in `masks/.zattrs`? If not, that would need adding.

Reply:
yes - that's what `ome-zarr` was using to load them into napari. This shows the 'split' masks as well as the 'labelled 0` mask:

```
$ cat 52.zarr/masks/.zattrs
{
    "masks": [
        "208516",
        "208517",
        "208518",
        "208519",
        "208520",
        "208525",
        "208526",
        "208527",
        "208528",
        "0"
    ]
```